app/config.py

```diff
@@ -7,7 +7,7 @@
     # from app.cloudfoundry_config import extract_cloudfoundry_config
     # extract_cloudfoundry_config()
     vcap_services = json.loads(os.environ['VCAP_SERVICES'])
-    os.environ['REDIS_URL'] = vcap_services['aws-elasticache-redis'][0]['credentials']['uri']
+    os.environ['REDIS_URL'] = vcap_services['aws-elasticache-redis'][0]['credentials']['uri'].replace("redis", "rediss")
```

Comment:
This is hacky. I'm going to try to convince CG to provide something like ['ssl-uri'] in the future.

Reply:
eh - not ideal but I'm not too worried about it. Another thing to potentially look into is if the library supports using the unmodified URL but with an `ssl` flag set to true. Or building the URL from the other data in the credentials hash.

Reply:
Yeah, I did try that, but unfortunately there's a limitation in the base redis-py lib that flaskredis wraps. The redis.from_url method does not accept ssl as a param the way that the client does when connecting with the traditional {host: x, port: x, pass: x} dict